lib/modules/@apostrophecms/piece-type/src/apos/components/AposPiecesManager.vue

```diff
@@ -0,0 +1,256 @@
+<template>
```

Comment:
These fully new files are almost all completely copied from storybook. I'll try to call out where they changed.